src/commands/account/aggregatebonded.ts

```diff
@@ -33,12 +33,12 @@ export default class extends AccountTransactionsCommand {
     @metadata
     execute(options: AccountTransactionsOptions) {
         this.spinner.start();
-        const profile = this.getProfile(options);
+        const profile = this.getProfile();
```

Comment:
I'm not sure we stop passing "options", could you clarify?

Reply:
I rewrited the function `getProfile()` in `profile.command.ts`.
Now, this function will get the default profile without any parameters.(command `nem2-cli profile current` will mark whitch profile is the default one)